app/controllers/transcripts_controller.rb

```diff
@@ -31,6 +31,10 @@ def search
   # GET /transcripts/the-uid
   # GET /transcripts/the-uid.json
   def show
+    if logged_in_user.admin? || logged_in_user.content_editor?
```

Comment:
Curious why `logged_in_user` is used here rather than the more standard devise `current_user` helper?

Reply:
On top of the `transcripts_controller.rb#logged_in_user` method there's this comment:
``` 
    # since we we using a combination of devise + rails and
    # API authenticatoin (with backbone in transcript edits page)
    # we need to check warden session here
    def logged_in_user
      warden.user
    end
``` 
I noticed that's what they're using so I'm using the same to keep it consistent.

Reply:
Fair enough :) Thought there was probably a reason. Seen enough people "re-inventing the wheel" to get suspicious ;)